studiopress/atomic-blocks
# Move assets to SP page builder directory

**Summary of change:**
Change image asset path from atomicblocks.com to demo.studiopress.com. 

**How to test:**
Check out the branch and check the first few sections and layouts to see if the images are loading from the new location. The rest of the sections and layouts will be addressed in the pro repo.

**Suggested Changelog Entry:**
Update path to remote assets.
